python/foglamp/services/common/microservice.py

```diff
@@ -90,3 +91,32 @@ def _get_service_registration_payload(self):
             }
         return service_registration_payload
 
+    @abstractmethod
+    async def shutdown(self):
+        pass
+
+    @abstractmethod
+    async def change(self):
+        pass
+
+    async def ping(self, request):
+        """ health check
+    
+        """
+        since_started = time.time() - self._start_time
+        return web.json_response({'uptime': since_started})
+
+    async def register(self, request):
```

Comment:
not implemented as they are part of non-core microservices, however they must exist for python to go forward